front/lib/plans/free_plans.ts

```diff
@@ -35,27 +51,26 @@ const FREE_PLANS_DATA: PlanAttributes[] = [
     isManagedGithubAllowed: true,
     maxNbStaticDataSources: -1,
     maxNbStaticDocuments: -1,
-    maxSizeStaticDataSources: -1,
+    maxSizeStaticDataSources: 2, // 2MB
```

Comment:
It's a big ask because it's a bit late :) But the fact that you comment 2MB here is probably a consequence of the name not specifying its units :)

Should probably be `maxSizeMbStaticDataSoruces` ?

Also this size applies to all data sources so I thiiink we don't want static in the name here?

As a consequence I thiiiiiink it's worth renaming :grimacing:

Reply:
Can be handled in a separate PR of course

Reply:
Yes sure, now is a good time to make sure we have proper naming; 
Will do in another PR to avoid another monster PR, adding a small card assigned to me, will do once this is merged 👍🏻